storage/migrations/mysql/6_subscriber_metrics.sql

```diff
@@ -0,0 +1,25 @@
+-- +migrate Up
+
+CREATE TABLE IF NOT EXISTS `processes` (
+    `id`                INTEGER UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
+    `name`              VARCHAR(30) UNIQUE NOT NULL,
+    `last_processed_id` BIGINT UNSIGNED NOT NULL,
+    `created_at`        datetime(6)            NOT NULL,
+    `updated_at`        datetime(6)            NOT NULL,
+) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+CREATE TABLE IF NOT EXISTS `accumulated_subscribes` (
```

Comment:
Can we clarify this tables values what will they represent? I'm guessing we will have a separate table for the cumulative values correct?

Reply:
At first, I thought since these records are daily we can use the same table to count each type daily (the three columns created, unsubscribed, and deleted would represent the daily number of events by type) and the so-called total column would represent the accumulated number of events. If you have something better shoot.